xbout/region.py

```diff
@@ -0,0 +1,589 @@
+from collections import OrderedDict
```

Comment:
We didn't need `OrderedDict`. I think I put them in because at one point I was considering accessing them by integer index sometimes, but didn't in the end. Will remove in this PR rather than #107 to avoid merge conflicts.